code/modules/antagonists/clock_cult/items/clockwork_item.dm

```diff
@@ -14,3 +14,5 @@
 		. += clockwork_desc
 	else if(desc)
 		. += desc
+
+	SEND_SIGNAL(src, COMSIG_PARENT_EXAMINE, user, .)
```

Comment:
What? Why there?

Reply:
just call parent

Reply:
The whole point is not calling parent to obscure the real description of the item.